Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -244,6 +244,21 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
         settingsCoordinator.start(with: section)
     }
 
+    private func showLibary(with homepanelSection: Route.HomepanelSection) {
+        let navigationController = DismissableNavigationViewController()
+        navigationController.modalPresentationStyle = .formSheet
+        let libraryRouter = DefaultRouter(navigationController: navigationController)
+
+        let libraryCoordinator = LibraryCoordinator(router: libraryRouter)
+        libraryCoordinator.parentCoordinator = self
+        add(child: libraryCoordinator)
+        libraryCoordinator.start(with: homepanelSection)
+
+        router.present(navigationController) { [weak self] in
+            self?.didFinishLibrary(from: libraryCoordinator)
+        }
```

Comment:
FYI In the case of settings page if I dismissed clicking the done button then this closure would not be called. We can ensure both cases properly deinit coordinator with https://github.com/mozilla-mobile/firefox-ios/issues/14779